File: projet-ebanking-EbankingDevFront/projet-ebanking-EbankingDevFront/src/app/model/dto/MaybeClientWithEmailTokenDTO.ts
export class MaybeClientWithEmailTokenDTO {
    firstName!: string;
    lastName!: string;
    email!: string;
    emailToken!: string;

    constructor(firstName: string, lastName: string, email: string, emailToken: string) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.email = email;
    this.emailToken = emailToken;
  }
  }

File: projet-ebanking-ebankingSecuredDev/projet-ebanking-ebankingSecuredDev/src/main/java/com/ebank/ebanking2/model/dto/MaybeClientWithEmailTokenDTO.java
package com.ebank.ebanking2.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MaybeClientWithEmailTokenDTO {
    private String firstName;
    private String lastName;
    private String email;
    private String emailToken;
}
